src/spikeinterface/preprocessing/filter.py

```diff
@@ -30,28 +30,30 @@ class FilterRecording(BasePreprocessor):
 
     Parameters
     ----------
-    recording : Recording
+    recording: Recording
         The recording extractor to be re-referenced
-    band : float or list, default: [300.0, 6000.0]
+    band: float or list, default: [300.0, 6000.0]
         If float, cutoff frequency in Hz for "highpass" filter type
         If list. band (low, high) in Hz for "bandpass" filter type
-    btype : "bandpass" | "highpass", default: "bandpass"
+    btype: "bandpass" | "highpass", default: "bandpass"
         Type of the filter
-    margin_ms : float, default: 5.0
+    margin_ms: float, default: 5.0
         Margin in ms on border to avoid border effect
-    filter_mode : "sos" | "ba", default: "sos"
+    filter_mode: "sos" | "ba", default: "sos"
         Filter form of the filter coefficients:
         - second-order sections ("sos")
-        - numerator/denominator : ("ba")
-    coef : array or None, default: None
+        - numerator/denominator: ("ba")
+    coef: array or None, default: None
         Filter coefficients in the filter_mode form.
-    dtype : dtype or None, default: None
+    dtype: dtype or None, default: None
         The dtype of the returned traces. If None, the dtype of the parent recording is used
+    causal: True or False, default: False
+        if True, backward causal filtering is performed to correct hardware induced phase shift
```

Comment:
maybe it would be better to have `causal_mode`:
- `filtfilt`
- `forward`
- `backward`
?